Stock Photos: Tracks events

ctarda opened the pull request:
Implements part of #8954 , Tracks events

I am not sure, in particular, of the way I am tracking the upload event. 

To test:
- Access Stock Photos, from the Media Library and Aztec editor. Confirm an event is tracked.
- Pick a photo from Stock Photos, it will be uploaded to the Media Library. Confirm an event is tracked.
- Insert a Stock Photo directly into the editor. An event should be tracked as well.

etoledom commented on WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift:
```diff
@@ -647,7 +647,9 @@ extension MediaLibraryViewController: StockPhotosPickerDelegate {
-            mediaCoordinator.addMedia(from: $0, to: blog)
+            let info = MediaAnalyticsInfo(origin: .editor(.stockPhotos), selectionMethod: .fullScreenPicker)
+            mediaCoordinator.addMedia(from: $0, to: blog, analyticsInfo: info)
+            WPAnalytics.track(.stockMediaUploaded)
```
This one (`WPAnalytics.track(.stockMediaUploaded)`) looks fine there!

The previous comment was about `MediaAnalyticsInfo(origin: .editor(.stockPhotos), selectionMethod: .fullScreenPicker)`.

-- Commenting this just to avoid confusion --

ctarda commented:
Thanks for clarifying @etoledom. I have removed the redundant event, and added the one missing (search). I will look into adding support for stock photos as a source of `.mediaLibrary` as you mentioned
